src/basho_bench_duration.erl

```diff
@@ -67,8 +67,8 @@ handle_call(remaining, _From, State) ->
     maybe_end({reply, Remaining, State}).
 
 
-handle_cast({worker_stopping, WorkerPid}, State) -> 
-    case basho_bench_worker_sup:active_workers() -- [WorkerPid] of 
+handle_cast({worker_stopping, WorkerId}, State) ->
+    case basho_bench_worker_sup:active_workers() -- [WorkerId] of
```

Comment:
Find that the application can not be stopped in sometimes.
After looking into the basho_bench code, I found that the WorkerPid is the driver pid in actually(it is the pid of replication driver in this case, ``` WorkerPid = spawn_link(fun() -> worker_init(State) end), WorkerPid ! {init_driver, self()},```). But the pid list which returned from basho_bench_worker_sup:active_worker/1 is basho_bench_worker's.  So if driver has been stopped but worker hasn't been stopped very soon, then ```basho_bench_worker_sup:active_workers() -- [WorkerPid]``` will not return [], it will return worker process id.
Only if the worker has also been stopped very soon, then ```[] -- [WorkerPid]``` can still return []. That's  why the application can be stopped normally in sometimes and sometimes not.

Now I change it to ```case basho_bench_worker_sup:active_workers() -- [WorkerId] ```, the WorkerId is basho_bench_worker id, then no matter whether the worker stop very soon or not, then the app can stop normally.